pkg/connector/tickets.go

```diff
@@ -27,14 +29,67 @@ type TicketManager interface {
 	ListTicketSchemas(ctx context.Context, pToken *pagination.Token) ([]*v2.TicketSchema, string, annotations.Annotations, error)
 }
 
-func getJiraStatusesForProject(ctx context.Context, client *jira.Client, projectId string) ([]jira.JiraStatus, error) {
+// example https://developer.atlassian.com/server/jira/platform/jira-rest-api-example-create-issue-7897248/
+func (j *Jira) customFieldSchemaToMetaField(field *v2.TicketCustomField) (interface{}, error) {
+	if field == nil {
+		return nil, nil
+	}
+
+	type JiraPickerStruct struct {
+		Id string `json:"id"`
+	}
+
+	pickObjects := []*JiraPickerStruct{}
```

Comment:
This could be set in the scope it's used after line 72. Probably not valuable to have it outside that case statement.